### PR TITLE
ci: ignore typos in string literals

### PR DESCRIPTION
**Pull request by Young-Flash**
cc @hackwaly

### CODE REVIEW

**Review by lijunchen — APPROVED** [review]
bleeding CI failed as expected

File: _typos.toml [hunk 1]
```diff
@@ -4,3 +4,8 @@ extend-exclude = ["**/*_gen.mbt"]
 [default.extend-identifiers]
 IDentifier = "IDentifier"
 
+[default]
+extend-ignore-re = [
+    # ignore string literals
+    "\"(.*?)\"",
+]
```
Comment (bobzhang) [inline _typos.toml:11]: also need support `#|` style string literals

**Comment by hackwaly** [comment]
Need a a test to verify it.

**Comment by Young-Flash** [comment]
I have test locally, should we need to add some intentional typos in the codebase to verify?

if so, which file would be appropriate to add these typos to?

**Comment by hackwaly** [comment]
> I have test locally, should we need to add some intentional typos in the codebase to verify?
> 
> if so, which file would be appropriate to add these typos to?

If locally tested, I thinks it's fine to without test here. The coming MR has few cases of that.

**Pull request merged**